client/src/components/EmailAlertSignup.tsx

```diff
@@ -51,10 +51,14 @@ const BuildingSubscribeWithoutI18n = (props: BuildingSubscribeProps) => {
   const showSubscribed =
     (justSubscribed && !!user?.verified) || !!user?.subscriptions?.find((s) => s.bbl === addr.bbl);
 
+  const eventUserParams = { user_id: user?.id, user_type: user?.type };
+
   const [isEmailResent, setIsEmailResent] = React.useState(false);
   const [showSubscriptionLimitModal, setShowSubscriptionLimitModal] = useState(false);
 
   const navigateToLogin = () => {
+    // logAmplitudeEvent("registerLoginViaBuilding");
```

Comment:
do we want to remove the amplitude event logs altogether? if I recall correctly, we haven't been tracking the login related events on amplitude thus far anyway

Reply:
yeah lets just not bother with it - removed all the comments